dask_expr/_collection.py

```diff
@@ -695,14 +700,20 @@ def prod(self, skipna=True, numeric_only=False, min_count=0, split_every=False):
             self.expr.prod(skipna, numeric_only, min_count, split_every)
         )
 
-    def var(self, axis=0, skipna=True, ddof=1, numeric_only=False):
-        return new_collection(self.expr.var(axis, skipna, ddof, numeric_only))
+    def var(self, axis=0, skipna=True, ddof=1, numeric_only=False, split_every=False):
+        return new_collection(
+            self.expr.var(axis, skipna, ddof, numeric_only, split_every=split_every)
+        )
 
-    def std(self, axis=0, skipna=True, ddof=1, numeric_only=False):
-        return new_collection(self.expr.std(axis, skipna, ddof, numeric_only))
+    def std(self, axis=0, skipna=True, ddof=1, numeric_only=False, split_every=False):
+        return new_collection(
+            self.expr.std(axis, skipna, ddof, numeric_only, split_every=split_every)
+        )
 
-    def mean(self, skipna=True, numeric_only=False, min_count=0):
-        return new_collection(self.expr.mean(skipna, numeric_only))
+    def mean(self, skipna=True, numeric_only=False, min_count=0, split_every=False):
+        return new_collection(
+            self.expr.mean(skipna, numeric_only, split_every=split_every)
+        )
```

Comment:
Does this close https://github.com/dask-contrib/dask-expr/issues/501 then?

Reply:
Probably, will close for now